auth-api/src/auth_api/models/entity.py

```diff
@@ -37,7 +37,7 @@ class Entity(BaseModel):  # pylint: disable=too-few-public-methods, too-many-ins
     pass_code_claimed = Column('pass_code_claimed', Boolean(), default=False)
     business_number = Column('business_number', String(100), nullable=True)
     name = Column('name', String(250), nullable=True)
-    corp_type_code = Column(String(10), ForeignKey('corp_type.code'), nullable=False)
+    corp_type_code = Column(String(15), ForeignKey('corp_type.code'), nullable=False)
```

Comment:
this change is intentional rite? from column length 10 to 15?

Reply:
Please see comments below, base model was changed earlier but not all references were not changed.